f3/granite.go

```diff
@@ -608,21 +520,21 @@ type quorumState struct {
 	// The power supporting each chain so far.
 	chainPower map[TipSetID]chainPower
 	// Total power of all distinct senders from which some chain has been received so far.
-	sendersTotalPower uint
+	sendersTotalPower StoragePower
```

Comment:
It's tricky to know how to reference Go big.Ints properly. Is there a good reason this one is not a pointer but the others are? Using a pointer seems to be canonical so I'd suggest doing that here too, unless there's a good reason not to.

Reply:
You are absolutely right. I started with no pointers but missed this one when switching to pointers.

Reply:
Done